UNRELEASED.md

```diff
@@ -4,6 +4,7 @@
 
 ### Enhancements
 
+- Removed customer images from avatar and updated to use new color system [#2449](https://github.com/Shopify/polaris-react/pull/2449)
```

Comment:
Should customer images be removed in current Polaris, too? If that’s the case I think the customer images removal should be done in its own PR. We also have not been adding changelog entries for color system changes since they are invisible unless the feature flag is turned on and somewhat unstable.

Reply:
I think it makes sense to remove them before the masterbrand changes as there's no way to fallback to them once the new icon is added. I'll move that to a new PR and it doesn't have to be merged right away